app/src/androidTest/java/com/android/sample/ui/dialogs/AddImageDialogTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.android.sample.ui.dialogs
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import org.junit.Rule
+import org.junit.Test
+
+class AddImageDialogTest {
+
+  @get:Rule val composeTestRule = createComposeRule()
+
+  @Test
+  fun addImageDialogDisplaysCorrectly() {
+    composeTestRule.setContent {
+      AddImageDialog(onDismiss = {}, onGalleryClick = {}, onCameraClick = {})
```

Comment:
Great. Here, instead of the other test file, you really call the code function, which is useful for testing.